### PR TITLE
Nodejs/week3

### PR DESCRIPTION
**Pull request by Deepti-Dk**

### CODE REVIEW

File: nodejs/week3/mealReservation/src/backend/api/meals.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+      meals = await knex('meal')
+        .select('*')
+        .where('created_date', '>', request.query.createdAfter.toString());
+    } else if (request.query.limit) {
+      //limit	Only specific number of meals	Number /meals?limit=4
+      meals = await knex('meal')
+        .select('*')
+        .orderBy('id')
+        .limit(Number(request.query.limit));
+    } else if (request.query.availableReservations) {
```
Comment (Laila1222) [inline nodejs/week3/mealReservation/src/backend/api/meals.js:34]: Try: `availableReservations === 'true'` here

**Review by Laila1222 — CHANGES_REQUESTED** [review]
Hi Deepti :) 
It seems that you have pushed the node_modules file, as well as the whole week1 and week2 folders. I suppose there might have been something with creating new branches. Did you create a new branch from master when you created nodejs/week3? If not, that could be one explanation for the issue. 
I'm writing this, because in the 'Changed files' section, we only want to see the files that you actually worked with for this homework.  
Check out these answers for what to do: https://stackoverflow.com/questions/1139762/ignore-files-that-have-already-been-committed-to-a-git-repository 

Otherwise, very good job on your homework! 🥇

**Comment by Deepti-Dk** [comment]
to be ignored as included all three branches HW for w1,w2,w3. will create a new PR just for w3

**Comment by Deepti-Dk** [comment]
Re-opening!!!hopefully just with w3 HW

**Comment by Deepti-Dk** [comment]
deleting it (will copy a new one)

**Pull request closed without merging**